demo/atom/validationText/playground

```diff
@@ -0,0 +1,26 @@
+const input = {
+  borderColor: '#bbbbbb',
+  borderStyle: 'solid',
+  borderWidth: '1px',
+  fontSize: '16px',
+  height: '40px',
+  paddingLeft: '16px',
+  paddingRight: '16px',
+  width: '100%',
+}
+return (
+  <div>
+    <div style={{marginBottom: '16px'}}>
+      <input style={input}
+        type='text'
+      />
+      <AtomValidationText type='success' text='Success validation text'/>
```

Comment:
it would be nice if you could export the validation types instead of using magic strings, like done here https://github.com/SUI-Components/sui-components/blob/master/demo/atom/spinner/playground#L58